src/MICore/LaunchOptions.cs

```diff
@@ -1877,6 +1871,8 @@ protected void InitializeCommonOptions(Xml.LaunchOptions.BaseLaunchOptions sourc
                     throw new InvalidLaunchOptionsException(MICoreResources.Error_InvalidSymbolInfo);
                 }
             }
+
+            this.Environment = new ReadOnlyCollection<EnvironmentEntry>(GetEnvironmentEntries(source.Environment) ?? new List<EnvironmentEntry>());
```

Comment:
?? will never return false since GetEnvironmentEntries seems to always return a non-null.